libs/hdf-converters/test/attestations/attestations.spec.ts

```diff
@@ -6,108 +6,371 @@ import {
   advanceDate,
   Attestation,
   createAttestationMessage,
-  parseXLSXAttestations
+  parseXLSXAttestations,
+  convertAttestationToSegment
 } from '../../src/utils/attestations';
 import {omitHDFTimes, omitVersions} from '../utils';
-
-const attestationArray: Attestation[] = [
+import _ from 'lodash';
+// test a new attestation array for each frequency, use arbitrary far date (yr 3000, 1900)
+// try applying to attestations w diff statuses
+const attestationMessageArray: Attestation[] = [
   {
     control_id: 'V-72087',
-    explanation:
-      'Audit logs are automatically backed up and cleared as necessary',
-    frequency: 'monthly',
+    explanation: 'This is a passing attestation that is not expired',
+    frequency: '1d',
     status: ExecJSON.ControlResultStatus.Passed,
-    updated: moment().format('YYYY[-]MM[-]DD'),
-    updated_by: 'Json Smith, Security'
+    updated: '3000-01-01',
+    updated_by: 'John Doe'
   },
   {
     control_id: 'V-73163',
-    explanation: 'Audit records are enabled for this system.',
+    explanation: 'This is a passing attestation that is expired',
+    frequency: '1d',
+    status: ExecJSON.ControlResultStatus.Passed,
+    updated: '1999-01-01',
+    updated_by: 'Jane Doe'
+  },
+  {
+    control_id: 'V-73164',
+    explanation: 'This is a failing attestation that is not expired',
+    frequency: '1d',
+    status: ExecJSON.ControlResultStatus.Failed,
+    updated: '3000-01-01',
+    updated_by: 'John Doe'
+  },
+  {
+    control_id: 'V-73165',
+    explanation: 'This is a failing attestation that is expired',
+    frequency: '1d',
+    status: ExecJSON.ControlResultStatus.Failed,
+    updated: '1999-01-01',
+    updated_by: 'Jane Doe'
+  }
+];
+
+const aPass_cSkip_unexp: Attestation[] = [
```

Comment:
```suggestion
const validPassingAttestationForSkippedControl: Attestation[] = [
```

I recommend a more spelled out name. I'm not worried about short variable names here. We have the real estate and it's easier to understand something more spelled out. The one I listed is only a suggestion. Feel free to modify as you choose. (In this example I used valid for not expired, and would have put expired for expired, but you can brainstorm that, too.)

Reply:
Sure. It does seem clearer to use valid/expired vs unexpired/expired since that's easier to distinguish while reading. I went with validPassingAttestation_passingControl with an underscore between the attestation and control descriptors